ee_core/src/main.c

```diff
@@ -121,10 +121,6 @@ static int eecoreInit(int argc, char **argv)
     CustomOSDConfigParam.timezoneOffset = _strtoi(_strtok(NULL, " "));
     enforceLanguage = _strtoi(_strtok(NULL, " "));
 
-    if (enforceLanguage)
```

Comment:
Why is this being removed?

Reply:
Because since we don't have libc on EE core there is no need to keep it

Reply:
oof lol sorry didn’t look at it properly, they’re all args of printf thought it was the actual setting state.. GitHub wordwraps shit on my phone all of a sudden makes it very annoying to read the code when skimming